File: RGBscript.m
R=imread('RGB photos/cubes_R.jpg');
G=imread('RGB photos/cubes_G.jpg');
B=imread('RGB photos/cubes_B.jpg');
C=imread('RGB photos/cubes_color.jpg');

r=R(:,:,1);
g=G(:,:,2);
b=B(:,:,3);

[filas columnas capas]=size(C);

I=uint8(zeros(filas, columnas, capas));

I(:,:,1)=r;
I(:,:,2)=g;
I(:,:,3)=b;

figure(1)
subplot(1,2,1)
imshow(C)
subplot(1,2,2)
imshow(I)

File: RGBscript.py
import matplotlib.pyplot as plt
import numpy as np


R=np.copy('cubes_R.jpg');
G=np.copy('cubes_G.jpg');
B=np.copy('cubes_B.jpg');
C=np.copy('cubes_color.jpg')

r=R[:,:,0];
g=G[:,:,1];
b=B[:,:,2];

# [filas columnas capas]=size(C);

filas,columnas,capas=C.shape

filas = C.shape[0]
columnas = C.shape[1]
capas = C.shape[2]

I=np.zeros((filas, columnas, capas),dtype=np.uint8)

I[:,:,0]=r
I(:,:,1]=g
I[:,:,2]=b
